### PR TITLE
fix(preset.style): font-family fallback

### PR DESCRIPTION
**Pull request by tenphi**
### Describe changes

Fix font family fallback issues.

##### Checklist

Before taking this PR from the draft, please, make sure you have done the following:

- [ ] Pipeline is passed
- [ ] Tests are added (including unit tests and stories in the storybook)
- [ ] Tests are passed successfully
- [ ] If you're adding a new component/new props, add stories that describe how this component/prop works
- [ ] Changeset(s) is(are) added
- [ ] You have passed the threshold of the library size
- [ ] Commit message follows [commit guidelines](https://github.com/cube-js/cube-ui-kit/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: .changeset/poor-rules-knock.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@cube-dev/ui-kit": patch
+---
+
+fix(preset.style): font-family fallback
```
Comment (MrFlashAccount) [inline .changeset/poor-rules-knock.md:5]: Please, don't write changelog as you write your commits, prefer a human-oriented style instead.

File: src/tasty/styles/preset.ts [hunk 2]
```diff
@@ -10,7 +10,17 @@ function setCSSValue(
   styles[`--${styleName}`] =
     presetName === 'inherit'
       ? 'inherit'
-      : `var(--${presetName}-${styleName}, var(--default-${styleName}, inherit))`;
+      : presetName === 'default'
```
Comment (MrFlashAccount) [inline src/tasty/styles/preset.ts:13]: I'm sure no one, even you after some period of time will never understand this mess of `?:${}()` symbols. Even if you wanna assign value where it's declared, I would suggest using inline self-calling arrow functions, e.g.

```typescript
styles[`--${styleName}`] = (() => {
  if (presetName === 'inherit') return 'inherit';
  // ...
})()
```

Comment (tenphi) [reply src/tasty/styles/preset.ts]: Fixed

**Review by MrFlashAccount — APPROVED** [review]
LGTM 🫶

**Pull request merged**